src/gui/src/displayControls.cpp

```diff
@@ -196,8 +196,8 @@ QVariant DisplayControlModel::data(const QModelIndex& index, int role) const
         // direction
         add_prop("Direction", information);
 
-        // min width
-        add_prop("Minimum width", information);
+        // min path width
+        add_prop("Minimum path width", information);
```

Comment:
What does 'path' imply here?

Reply:
that is what it's called in the ODB, so I copied it over

Reply:
I guess you mean this comment "Get the minimum path-width.".  I don't know why somebody wrote that as it applies to everything not just paths (cut layers have min width too).  Lets leave it as "Minimum width".   If you want to fix the comment in odb that would be better.

Reply:
So how does getWidth differ from getMinWidth?

Reply:
LEF has both WIDTH and MINWIDTH keywords.  All PDKs I looked at specified WIDTH.  Some also specified MINWIDTH but it was always equal to WIDTH.   Only one had both but with different values for certain layers.  

The LEF manual says of MINWIDTH "This value is used for verification purposes, and does not affect the routing width."    Whereas WIDTH is "Specifies the default routing width to use for all regular wiring on the layer. "

For further fun, on a cut layer there is only WIDTH which means "Specifies the minimum width of a cut."

Perhaps getWidth() should be "default width" since it may not be the minimum value.